lib/lackeys/rails_base.rb

```diff
@@ -37,5 +37,21 @@ def method_missing(method_name, *args, &block)
         super
       end
     end
+
+    #######################
+    # Development methods #
+    #######################
+    def who_has?(method_name)
+      return nil unless respond_to?(method_name)
```

Comment:
What about returning an empty array here? It might be more consistent with other results?